analysis/tests/workspace/expected/workspaceSymbol.txt

```diff
@@ -0,0 +1,1637 @@
+[{
```

Comment:
All symbols in `tests/src` dir

Reply:
This is going to be noisy when new test files are added. Also this output is big.
How about using a couple of small files and a subdirectory.

Reply:
This is the one thing still holding the PR from being merged.